### PR TITLE
Fix for jest handling of static assets when running tests. 

### PR DESCRIPTION
**Pull request by raythree**
Fix for jest handling static assets
https://github.com/facebook/jest/issues/2663#issuecomment-317109798

### CODE REVIEW

**Comment by coryhouse** [comment]
Thanks @raythree! 💯 

Guys, this look good to you?

**Comment by kwelch** [comment]
Added a few comments, the fix itself all appears to be valid but a few things related to clean up.

**Comment by coryhouse** [comment]
Thanks guys! Merging.

File: package.json [hunk 1]
```diff
@@ -112,10 +112,10 @@
     "url": "https://github.com/coryhouse/react-slingshot"
   },
   "jest": {
-    "moduleNameMapper": {
-      "\\.(css|scss)$": "identity-obj-proxy",
-      "^.+\\.(gif|ttf|eot|svg|woff|woff2|ico)$": "<rootDir>/tools/fileMock.js"
-    }
+    "moduleNameMapper": { 
+    "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tools/assetsTransformer.js", 
+    "\\.(css|less)$": "<rootDir>/tools/assetsTransformer.js"
```
Comment (coryhouse) [inline package.json]: Oops, good points that I overlooked @kwelch. @raythree?

Comment (kwelch) [inline package.json]: No worries I made these changes directly to master. The not regex did not work as I had intended.

**Pull request merged**